homeassistant/components/unifi/services.py

```diff
@@ -110,3 +115,37 @@ async def async_remove_clients(hass, data) -> None:
 
         if clients_to_remove:
             await controller.api.clients.remove_clients(macs=clients_to_remove)
+
+async def async_block_client(hass, data) -> None:
```

Comment:
Can you keep registering two services but have just one function for both services?